fendermint/app/options/src/genesis.rs

```diff
@@ -36,6 +40,26 @@ pub enum GenesisCommands {
     IntoTendermint(GenesisIntoTendermintArgs),
 }
 
+#[derive(Args, Debug)]
+pub struct GenesisSetEAMPermissionsArgs {
+    #[arg(
+        long,
+        short,
+        default_value = "unrestricted",
+        help = "Permission mode (unrestricted/allowlist) that controls who can deploy contracts in the subnet"
+    )]
+    pub mode: String,
```

Comment:
It wouldn't take long to create an enum with `ValueEnum` for this to get validation and help.

Reply:
I think that can be said for all other enum-based parameters. The CLI needs **a lot** of love all around, I wouldn't block this PR to seek perfectionism.